fiftyone/utils/sam2.py

```
@@ -32,89 +32,75 @@
-        )
-        if nx2 > nx1 and ny2 > ny1:
-            if mask.ndim == 2:
-                mask[ny1:ny2, nx1:nx2] = 0
-            elif mask.ndim == 3:
-                mask[:, ny1:ny2, nx1:nx2] = 0
-            else:
-                mask[:, 0, ny1:ny2, nx1:nx2] = 0
 
-    return mask
+class _SAM2Predictor(fosam._SAMPredictor):
+    def __init__(self, model):
```

Ruff (0.15.9)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)